INSTALL_MACOS.md

```diff
@@ -1,15 +1,14 @@
-Instructions for installing Natron from sources on OS X
-=======================================================
+# Instructions for installing Natron from sources on macOS
 
-These are step-by-step instructions to compile Natron on OS X.
+These are step-by-step instructions to compile Natron on macOS.
 
-OS X 10.6 (a.k.a. Snow Leopard) and newer are supported when building with MacPorts, and Homebrew can be used to compile it on the latest OS X.
+macOS 10.6 (a.k.a. Snow Leopard) and newer are supported when building with MacPorts, and Homebrew can be used to compile it on the latest version of macOS.
 
 ## Official Natron binaries
 
 The official Natron and plugins binaries are built using the section about MacPorts in these instructions to prepare the system, and the `launchBuildMain.sh` build script found in the `tools/jenkins` directory. The script takes care of everything, from checking out sources, to compiling and packaging.
 
-These binaries are built on an OS X 10.9 (Mavericks) virtual machine with [Xcode 6.2](https://developer.apple.com/devcenter/download.action?path=/Developer_Tools/Xcode_6.2/Xcode_6.2.dmg). Note that [Mavericks can not be downloaded anymore from the 10.14 (High Sierra) App Store](https://www.macworld.co.uk/how-to/mac-software/download-old-os-x-3629363/), so you will need to use a Mac with an older system (up to 10.13), or look for [alternatives](https://applehint.com/t/download-all-macos-x-10-4-10-14-original/376).
```

Comment:
Mavericks is actually OS X Mavericks, not macOS https://en.wikipedia.org/wiki/OS_X_Mavericks